xrayutilities/materials/elements.py

```diff
@@ -115,7 +118,7 @@
 Tc = Atom('Tc', 43)
 Ru = Atom('Ru', 44)
 Ru3p = Atom('Ru3+', 44)
-Rup4 = Atom('Ru+4', 44)
+Ru4p = Atom('Ru4+', 44)
 Rh = Atom('Rh', 45)
```

Comment:
clearly a bug on my side.... why you did not send these things upstream much earlier?

Reply:
No good reason... :-)